demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
```

Comment:
I believe this must appear on the first line or the shebang has no effect and will probably use the default interpreter, mostly likely `sh`.

https://en.wikipedia.org/wiki/Shebang_(Unix)